src/lbaf/Model/lbsObject.py

```diff
@@ -3,25 +3,49 @@
 from .lbsBlock import Block
 from .lbsObjectCommunicator import ObjectCommunicator
 
-
 class Object:
-    """A class representing an object with load and communicator"""
+    """A class representing an object with load and communicator
+
+    Constructor arguments:
+    :arg seq_id: small int identifier, commonly the task index, defaults to None
+    :arg packed_id: alt identifier (generated by vt as a bitpacked value), defaults to None
+    :arg r_id: rank ID, defaults to None
+    :arg load: the computational time, also known as load, defaults to 0.0
+    :arg size: the size, defaults to 0.0
+    :arg comm: the communicator, defaults to None
+    :arg user_defined: user defined data dict, defaults to None
+    :arg subphases: list of subphases, defaults to None
+    :arg collection_id: collection id (required for migratable objects)
+    """
 
     def __init__(
         self,
-        i: int,
+        seq_id: Optional[int] = None,
+        packed_id: Optional[int] = None,
         r_id: Optional[int]=None,
         load: float=0.0,
         size: float=0.0,
         comm: Optional[ObjectCommunicator]=None,
         user_defined: dict=None,
-        subphases: list=None):
+        subphases: list=None,
+        collection_id: Optional[int] = None):
+
+        # Check that id is provided as defined in LBDatafile schema
+        if packed_id is None and seq_id is None:
+            raise ValueError('Either packed_id (bit-encoded ID) or i (seq ID) must be provided.')
```

Comment:
I think this might be a typo: `... or i (seq ID) ...`

Reply:
Fixed